src/components/applicant-profile/GrantsList.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Paper from "@material-ui/core/Paper";
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import { useStyles } from "./GrantsList.styles";
+
+export default function GrantsList() {
+  const classes = useStyles();
+
+  // const dispatch = useDispatch();
+  // const user = useSelector((state) => state.login.user)
+
+  // useEffect(() => {
```

Comment:
I'm assuming this code is commented out because it will be needed once we hook up the back end?